tutorials/spring-boot-restful/src/main/kotlin/org/jetbrains/kotlin/demo/Application.kt

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.kotlin.demo
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+
+@SpringBootApplication
+class Application {
```

Comment:
I suggest adding a comment like `// your @Bean declarations here` then. With Kotlin's minimalism approach it metaphorically burns the eyes when something unneeded is put into code.